File: server/test-data/symbol/java/project1/src/org/DocumentSymbol.java
package org;

import com.Class1;

public class DocumentSymbol extends BaseClass implements Interface {
  private int x;
  private static final Class1 cls = new Class1();

  public DocumentSymbol(int x) {
    this.x = x;
  }

  @Override
  public int foo(int x, String str) {
    int a = 1;
    String project = "lsp";
    Class cls = null;
    Boolean b = true;
    func();
  }

  public interface EntryInter {
    void foo();
  }

  public enum Litter {
    A, B;
  }
}

File: server/test-data/symbol/kotlin/project1/src/org/DocumentSymbol.kt
package org

import com.Class1

enum class Letter {
  A, B
}

interface Interface {
  fun foo(x: Int, str: String): Int;
}

@Special("for test")
open class DocumentSymbol(x: Int) : BaseClass(x), Interface {
  private var x = x;
  private val cls = Class1();

  override fun foo(x: Int, str: String): Int {
    val a = 1
    val cls = null
    val b = true
    return func()
  }

  fun bar(): Int = 42
}

fun buz(a: Int): Int = a + 1